Documentation/ceph-cluster-crd.md

```diff
@@ -8,8 +8,11 @@ indent: true
 Rook allows creation and customization of storage clusters through the custom resource definitions (CRDs).
 
 ## Sample
+
 To get you started, here is a simple example of a CRD to configure a Ceph cluster with all nodes and all devices. More examples are included [later in this doc](#samples).
 
+**NOTE** In addition to your CephCluster object, you need to create RBAC rules for the namespace you are going to create the CephCluster in, see [Common Cluster Resources](#common-cluster-resources) section below.
```

Comment:
What if you put all these **NOTE** sections below the yaml snippets? It's nice to see the sample before being told what to worry about. :)

Reply:
I would like to keep the `NOTE` sections before the snippets so users see them first, think and then "just" copy'n'paste.